sdk/data/azcosmos/cosmos_container.go

```diff
@@ -358,6 +359,226 @@ func (c *ContainerClient) ReadItem(
 	return newItemResponse(azResponse)
 }
 
+// PatchItem patches an item in a Cosmos container.
+// ctx - The context for the request.
+// partitionKey - The partition key for the item.
+// itemId - The id of the item to patch.
+// item - The content to be used to patch.
+// o - Options for the operation.
+func (c *ContainerClient) PatchItem(
```

Comment:
Once we land on the final public API, adding one example to https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/data/azcosmos/example_test.go named `ExampleContainerClient_PatchItem` would be great as it would appear in the go pkg docs

Reply:
Done